securedrop_export/disk/cli.py

```diff
@@ -388,7 +551,7 @@ def _unmount_volume(self, volume: MountedVolume) -> Volume:
         if os.path.exists(volume.mountpoint):
             logger.debug(f"Unmounting drive from {volume.mountpoint}")
             try:
-                subprocess.check_call(["sudo", "umount", volume.mountpoint])
+                subprocess.check_call(["udisksctl", "unmount", volume.mountpoint])
```

Comment:
`udisksctl unmount` appears to want `--block-device`, without which it errors out with its usage statement, so:

```suggestion
                subprocess.check_call(["udisksctl", "unmount", "--block-device", f"/dev/mapper/{volume.mapped_name}"])
```